extensions/markdown-language-features/src/util/topmostLineMonitor.ts

```diff
@@ -22,11 +35,50 @@ export class TopmostLineMonitor extends Disposable {
 				}
 			}
 		}));
+
+		this._register(vscode.window.onDidChangeActiveTextEditor(textEditor => {
+
+			// When at a markdown file, apply existing scroll settings from static preview if last editor was custom.
+			// Also save reference to text editor for line number reference later
+			if (textEditor && textEditor.document && isMarkdownFile(textEditor.document)) {
+				if (this.isPrevEditorCustom) {
+					const line = this.getPreviousStaticEditorLineByUri(textEditor.document.uri);
+					if (line) {
+						this._onEditorNeedsScrolling.fire({ line: line, editor: textEditor });
```

Comment:
Would it make sense to have the previewManager instead monitor `onDidChangeActiveTextEditor` and then ask the `TopMostLineMonitor` if it need to scroll somewhere?

Reply:
Sure, that would also functionally work. I can make that change.